### PR TITLE
chore(weave): Update Scorers tab to match design

### PR DESCRIPTION
**Pull request by tssweeney**
Updates:
1. Header tabs with icons
2. Common create form
3. Common infra

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx [hunk 1]
```diff
@@ -282,6 +295,7 @@ export const SimplePageLayoutWithHeader: FC<{
                         key={tab.label}
                         value={tab.label}
                         className="h-[30px] text-sm">
+                        {tab.icon ? <Icon name={tab.icon} /> : null}
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:298]:
```suggestion
                        {tab.icon && <Icon name={tab.icon} />}
```

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ScorersPage.tsx [hunk 2]
```diff
@@ -1,32 +1,66 @@
...
+            onClick={() => setIsModalOpen(true)}
+            variant="secondary">
+            Create scorer
+          </Button>
+        }
+        headerContent={undefined}
+        notifySelectedTab={tab =>
+          setSelectedTab(
+            Object.values(scorerTypeRecord).find(t => t.label === tab)?.value ??
+              'ANNOTATION'
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ScorersPage.tsx:55]: cant we use the constant type?

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx [hunk 3]
```diff
@@ -163,20 +165,29 @@ export const SimplePageLayoutWithHeader: FC<{
...
     label: string;
+    icon?: IconName;
     content: ReactNode;
   }>;
   headerExtra?: ReactNode;
   headerContent: ReactNode;
   leftSidebar?: ReactNode;
   hideTabsIfSingle?: boolean;
   isSidebarOpen?: boolean;
+  notifySelectedTab?: (tab: string) => void;
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:176]:
```suggestion
  onTabSelectedCallback?: (tab: string) => void;
```

**Review by gtarpenning — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ScorersPage.tsx (hunk 2 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ScorersPage.tsx:55]: yup, missed it

Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ScorersPage.tsx:55]: fixed in next commit

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx (hunk 3 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:176]: fixed in next commit

**Pull request merged**